Tasks/IISWebAppDeployment/task.json

```diff
@@ -150,6 +150,16 @@
             "helpMarkDown": "Select the option to add port binding for the website."
         },
         {
+            "name": "AssignDuplicateBinding",
+            "type": "boolean",
+            "label": "Assign Duplicate Binding",
+            "required": false,
+            "groupName": "website",
+            "defaultValue": "true",
+            "visibleRule": "AddBinding = true",
+            "helpMarkDown": "If selected, assign the binding to the website even if the binding is assigned to another site. If the binding is assigned to two sites,then only one will start."
```

Comment:
This appears little confusing. If only one site will start, then what is the scenario we are lighting up by this option?